lua/nvim_lsp/julials.lua

```diff
@@ -22,23 +32,24 @@ configs.julials = {
 https://github.com/julia-vscode/julia-vscode
 `LanguageServer.jl` can be installed via `:LspInstall julials` or by yourself the `julia` and `Pkg`:
 ```sh
-julia -e 'using Pkg; Pkg.add("LanguageServer")'
+julia --project=]] .. environment_directory .. [[ -e 'using Pkg; Pkg.add("LanguageServer"); Pkg.add("SymbolServer")'
```

Comment:
https://github.com/neovim/nvim-lsp/pull/258/files#diff-04c6e90faac2675aa89e2176d2eec7d8R2236

The document will be generated based on the environment in which the generating script was executed.
Since every user's environment is different, you have to ask the user ":lua print(require'nvim _lsp'.util.path.join(require'nvim_lsp'.util .base_install_dir, "julials"))" to check the path Encourage them to do so.

Reply:
I added a comment in this description guiding users that want to install it manually.